config/suit/templates/nrf54h20/default/v1/root_with_binary_nordic_top.yaml.jinja2

```diff
@@ -185,77 +242,202 @@ SUIT_Envelope_Tagged:
 {%- endif %}
 
     suit-candidate-verification:
-    - suit-directive-set-component-index: 0
-{%- if radio is defined %}
+    - suit-directive-set-component-index: {{ top_component_index }}
     - suit-directive-override-parameters:
-        suit-parameter-uri: '#{{ radio['name'] }}'
-        suit-parameter-image-digest:
-          suit-digest-algorithm-id: cose-alg-sha-256
-          suit-digest-bytes:
-            envelope: {{ artifacts_folder ~ radio['name'] }}.suit
-    - suit-directive-fetch:
+        suit-parameter-version:
+          suit-condition-version-comparison-greater-equal: 0.8.0
```

Comment:
It is quite difficult to understand whether this semver refers to installed or candidate Nordic Top. It is worth to document (comment) it.

Reply:
Added a comment